docusaurus/docs/components/feedback/form.md

```diff
@@ -34,6 +34,10 @@ Array of Objects containing **`icon`** (String), **`description`** (String), and
 
 Array of Objects containing **`value`** (String,Number) and **`label`** (String) properties. Allows a dropdown displaying the options provided to let the user indicate what the feedback is about.
 
+#### `aboutPlaceholder?: string`
```

Comment:
type s/b `node`

Reply:
let's use the same convention for how the default value is represented for `prompt`